frontends/PyCDE/test/test_import_hw_modules.py

```diff
@@ -0,0 +1,35 @@
+# RUN: %PYTHON% py-split-input-file.py %s | FileCheck %s
+
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+
+from pycde.system import System
+
+tmp = NamedTemporaryFile()
+tmp.write(b"""
+hw.module @add(%a: i1, %b: i1) -> (out: i1) {
+  %0 = comb.add %a, %b : i1
+  hw.output %0 : i1
+}
+
+hw.module @and(%a: i1, %b: i1) -> (out: i1) {
+  %0 = comb.and %a, %b : i1
+  hw.output %0 : i1
+}
+""")
+tmp.flush()
+
```

Comment:
Can you add a PyCDE module interacting with a loaded one? Like instantiating one? I think you need to load the symbol into a namespace of some sort.

Reply:
This is what i'm working on now, and will include an update in this PR. I'm realizing that we'd want to use this a little differently, and that will probably be a change in the System API in this PR based on that usage.

Reply:
Yeah, this works as a base layer. I suspect we're gonna want some convenience functions on top for common use cases.